src/lang/en.json

```diff
@@ -588,7 +588,7 @@
     "notificationDescription": "Notifications must be assigned to a monitor to function.",
     "keywordDescription": "Search keyword in plain HTML or JSON response. The search is case-sensitive.",
     "invertKeywordDescription": "Look for the keyword to be absent rather than present.",
-    "jsonQueryDescription": "Do a json Query against the response and check for expected value (Return value will get converted into string for comparison). Check out {0} for the documentation about the query language. A playground can be found {1}.",
+    "jsonQueryDescription": "Parse and extract specific data from the server's JSON response using JSON query or use \"$\" for the raw response, if not expecting JSON. The result is then compared to the expected value, as strings. See {0} for documentation and use {1} to experiment with queries.",
```

Comment:
We might want to remove the other translations.
This is a significantly different (read: better) message and needs to be re-translated anyway.

Reply:
Thanks, I'll add remove the other translations for this PR, and commit.

Reply:
Resolved with a1f31f93736dedb29f85a6f7d67e31283c22767b